Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -835,8 +898,149 @@ def find_submission_id_command(
     return tableToMarkdown("CrowdStrike Falcon X response:", filtered_outputs), entry_context, [response]
 
 
+def get_results_function_args(outputs, extended_data, item_type, interval_in_secs):
+    if isinstance(outputs, list):
+        outputs = outputs[0]
+
+    results_function_args: dict = {
+        'ids': outputs.get('submitted_id'),
+        'extended_data': extended_data,
+        'interval_in_seconds': interval_in_secs,
+        'polling': True,
+    }
+    if item_type == 'FILE':
+        results_function_args['submit_file'] = 'yes'
+
+    return results_function_args
+
+
+def pop_polling_related_args(args):
+    if 'submit_file' in args:
+        args.pop('submit_file')
+    if 'enable_tor' in args:
+        args.pop('enable_tor')
+    if 'interval_in_seconds' in args:
+        args.pop('interval_in_seconds')
+    if 'polling' in args:
+        args.pop('polling')
+
+
+def should_run_upload_function(args):
+    return not args.get('ids')
```

Comment:
Let's change the name + add a docstring explaining why this means what it means

Reply:
done